third-party/qthread/qthread-1.10/src/interfaces/chapel/tasks-qthreads.c

```diff
@@ -615,13 +618,125 @@ void chpl_task_exit(void)
 #endif /* QTHREAD_MULTINODE */
 }
 
+//
+// Tasking callback support.
```

Comment:
Is there no way to share the implementation of install/uninstall/do_callback between tasking implementations? 

Would it be easier if the shim layer was moved into our runtime?